open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataTopicConnectorAuditCode.java

```diff
@@ -134,8 +128,13 @@
                     "indicate the cause of this error.  Work to clear the underlying error.  " +
                     "Once fixed, it may be necessary to restart the server to cause a reconnect to Kafka."),
 
-    ;
+    KAFKA_PRODUCER_SEND_ERROR( "OCF-KAFKA-TOPIC-CONNECTOR-0019" ,
+            OMRSAuditLogRecordSeverity.EXCEPTION,
+            " Egeria has encountered an unrecoverable error while attempting to send messages to the following topic {0}",
+            "Ensure that the kakfa service is available",
+            "Ensure that the kafka service is available"),
```

Comment:
For the user action, do they need to do anything to server to restart sends once Kafka is available?